plugins/modules/nd_setup.py

```diff
@@ -407,7 +454,7 @@ def main():
                             "keyID": server.get("ntp_key_id"),
                             "prefer": server.get("preferred"),
                         }
-                        for server in ([] if ntp_config is None else ntp_config.get("servers"))
+                        for server in (ntp_config.get("servers") if ntp_config is not None and ntp_config.get("servers") is not None else [])
```

Comment:
Any reason for changing this?

Reply:
Actually no reason, I just forgot that "servers" was a required attribute when "ntp_config" is used. So the code did not need to change.